src/lib/draw/HoverLayer.svelte

```diff
@@ -54,13 +61,15 @@
     "interventions-points"
   );
 
-  currentHover.subscribe((id) => {
+  // When a form is open, ignore regular map and sidebar interactions
```

Comment:
I feel like this is a simplification -- let's not abuse what object is hovered from the map, sidebar, etc. Calculate all of those things in a straightforward way. Then when we need to make a decision based on multiple values, we can just combine them